src/steps/05-PerformanceRequirements/DOW/ComputeForm.spec.ts

```diff
@@ -0,0 +1,700 @@
+        computeData.classificationLevel = 'class3';
+        await wrapper.setData({
+          allClassificationLevels: allClassificationLevels,
+          modalSelectedOptions: ["class4", "class5"]
+        })
+
+        //closes modal and sets avlClassificationLevelObjects with 
+        //class levels selected in modal
+        const classLevelsChangedSpy = await jest.spyOn(wrapper.vm, "classificationLevelsChanged");
+        await wrapper.vm.classificationLevelsChanged();
+        expect(classLevelsChangedSpy).toHaveBeenCalled();
+
+        // if the classification level that was selected was removed via the modal,
+        // reset computeData.classificationLevel
+        Vue.nextTick(() => {
+          expect(wrapper.vm.$props.computeData.classificationLevel).toBe("");
+        })
+      });
+    });
+
+    describe("Available Classification Levels Radio Group", () => {
+      it("tests if no items are loaded, then classificationRadioOptions array has length===0", async () => {
+        //set `avlClassificationLevelObjects` with data
+        await wrapper.setData({
+          avlClassificationLevelObjects: []
+        })
+        const spy = await jest.spyOn(wrapper.vm, "setAvlClassificationLevels");
+        await wrapper.vm.setAvlClassificationLevels();
+
+        // call `setAvlClassificationLevels` to transform data into 
+        // radio button list
+        expect(spy).toHaveBeenCalled();
+        Vue.nextTick(() => {
+          // ensure radio button list is the same length as array
+          expect(wrapper.vm.$data.classificationRadioOptions).toEqual(0);
+        })
+      });
+      it("-tests if populated successfully from `avlClassificationLevelObjects` array then " +
+         "classificationRadioOptions.length === avlClassificationLevelObjects.length", async () => {
+        //set `avlClassificationLevelObjects` with data
+        await wrapper.setData({
+          avlClassificationLevelObjects: avlClassificationLevelObjects
+        })
+        const spy = await jest.spyOn(wrapper.vm, "setAvlClassificationLevels");
+        await wrapper.vm.setAvlClassificationLevels();
+
+        // call `setAvlClassificationLevels` to transform data into 
+        // radio button list
+        expect(spy).toHaveBeenCalled();
+        Vue.nextTick(() => {
+          // ensure radio button list is the same length as array
+          expect(wrapper.vm.$data.classificationRadioOptions).toHaveLength(
+            avlClassificationLevelObjects.length
+          );
+        })
+      });
+
+    });
+
+    describe("Validation Tests...", () => {
+      describe("on page load and...", () => {
+        it("validates if page has been 'touched', if so, expect validate() to be called", async () => {
+          await wrapper.setData({
+            formHasBeenTouched: true
+          })
+          const spy = await jest.spyOn(wrapper.vm, "validate");
+          await wrapper.vm.validate();
+          expect(spy).toHaveBeenCalledTimes(1);
```

Comment:
This isn't actually testing anything because we're calling `validate()` ourselves

Reply:
all references to spy have been removed.